Assets/Scripts/PlayerController.cs

```diff
@@ -558,7 +558,7 @@ public void WinRumble()
         // This joystick - 1 shit has got to go.
         if (JoystickLayerManager.Instance != null && !isAI)
         {
-            JoystickLayerManager.Instance.ActivateLargeRumble(joystick - 1, 3f);
+            JoystickLayerManager.Instance.ActivateLargeRumble(joystick - 1, 2f);
```

Comment:
I didn't see this method, so neat!

Reply:
Yeah, check out the JoystickLayerManager and FilesystemLayer when you get a chance!